### ISSUE TITLE
Bug: 

### ISSUE THREAD

**Issue by apchirinoc** [nvim-lualine/lualine.nvim#762]
### Self Checks

- [x] I'm using the latest lualine.
- [x] I didn't find the issue in exsisting issues or prs.

### How to reproduce the problem
1. Install NeoVim 0.7.0
2. Install nvim-lsp and lualine with Packer
3. Install a language server
4. Configure lualine
5. Use `vim.lsp.buf.hover()` on a keyword in a file

### Expected behaviour
Floating window showing the hover info

### Actual behaviour
When the floating window shows up, the status bar disappears for a second and then it's there again

### Minimal config to reproduce the issue
`nvim --version` dump:
```
NVIM v0.7.0
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/gcc-11 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNVIM_TS_HAS_SET_MATCH_LIMIT -DNVIM_TS_HAS_SET_ALLOCATOR -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wdouble-promotion -Wmissing-noreturn -Wmissing-format-attribute -Wmissing-prototypes -Wimplicit-fallthrough -Wsuggest-attribute=pure -Wsuggest-attribute=const -Wsuggest-attribute=malloc -Wsuggest-attribute=cold -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/home/runner/work/neovim/neovim/build/config -I/home/runner/work/neovim/neovim/src -I/home/runner/work/neovim/neovim/.deps/usr/include -I/usr/include -I/home/runner/work/neovim/neovim/build/src/nvim/auto -I/home/runner/work/neovim/neovim/build/include
Compiled by runner@fv-az316-460

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "
/home/runner/work/neovim/neovim/build/nvim.AppDir/usr/share/nvim"
```

<details>
<summary>Config</summary>

</details>

**Comment by shadmansaleh** [comment]
Are you sure you updated lualine and on lateset version of lualine ?

Because this issue was fixed in 2d6108e07f5888a42dee37598570bac990ca1297 about an hour ago.

**Comment by apchirinoc** [comment]
@shadmansaleh Thank you for your answer. I don't know what exactly means an isolated minimal config, this is the first time I submit and issue in a plugin, I apologize.

I'm closing the issue as it was fixed in the commit you mentioned. Thanks again!
